.pre-commit-config.yaml

```diff
@@ -67,6 +67,8 @@ repos:
           - yarn.lock
           - --exclude-files
           - ".*/generated/"
+          - --exclude-files
+          - ".env.ci"
```

Comment:
IMO it would be preferable to use
```
DATABASE_URL=postgres://postgres:postgres@db:5432/postgres # pragma: allowlist secret
```
within `.env.ci`.

Reply:
Nice